autorest/adal/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Azure/go-autorest/autorest/adal
 
-go 1.12
+go 1.15
```

Comment:
This sets the minimum version to 1.15.  While we only officially support Go versions N and N-1, we try to keep things working on older versions where possible.

Reply:
Unfortunately, golang-jwt no longer with versions before 1.15; https://github.com/Azure/go-autorest/pull/645#issuecomment-898440791

Reply:
Note that (currently) the version in `go.mod` is only used as a "recommended' version; it's not (yet?) enforcing users to use this version, so depending on what code they consume, it may still work.

Reply:
@jhendrixMSFT ptal 🤗